include/zephyr/drivers/flash/stm32_flash_api_extensions.h

```diff
@@ -64,6 +64,23 @@ enum stm32_ex_ops {
 	FLASH_STM32_EX_OP_OPTB_WRITE,
 };
 
+enum stm32_qspi_ex_ops {
```

Comment:
Shouldn't it be wrapped within `#if defined(CONFIG_FLASH_EX_OP_ENABLED)`?

Reply:
In the same file, the other enum is not wrapped but I don't see any problem wrapping the one I added.